ui/src/router/details/logs/RouterLogsView.js

```diff
@@ -55,24 +67,73 @@ export const RouterLogsView = ({ router }) => {
     (entries) =>
       !!entries.length ? entries[entries.length - 1].timestamp : undefined,
     (entries) => entries.map((entry) => LogEntry.fromJson(entry).toString()),
-    configOptions
+      appConfig.podLogs,
   );
 
   const availableComponents = components.filter(
     (c) => c.value === "router" || !!get(router, `config.${c.value}`)
   );
 
+  useEffect(
+    () => {
+      let urls = {}
+      if (appConfig.imagebuilder.cluster && appConfig.imagebuilder.gcp_project && appConfig.imagebuilder.namespace &&
```

Comment:
nit: indent

Reply:
Ok turned it to following to make it clearer:
```js
      if (
          appConfig.imagebuilder.cluster &&
          appConfig.imagebuilder.gcp_project &&
          appConfig.imagebuilder.namespace &&
          currentProject
      ) {
```